src/widgets/File/File.jsx

```diff
@@ -85,11 +85,16 @@ class File extends Component {
           value: location
         });
 
-        if (this.props.registry.formContext && this.props.registry.formContext.setResourceSpecs) {
-          this.props.registry.formContext.setResourceSpecs({
+        const { formContext } = this.props.registry;
+        if (formContext && formContext.setResourceSpecs) {
+          formContext.setResourceSpecs({
             'dcat:byteSize': this.state.total,
             'dcat:mediaType': files[0].type,
-            'ams:distributionType': 'file'
+            'ams:distributionType': 'file',
+            'dct:modified': new Date().toISOString().split('T')[0],
```

Comment:
Why not have the same as getNow() in src/fields/Resources/Resources.jsx?

Reply:
Thanks, I wanted to move that to a service already